hilti/runtime/include/global-state.h

```diff
@@ -69,6 +70,12 @@ struct GlobalState {
      * still need this information.
      */
     std::vector<hilti::rt::detail::HiltiModule> hilti_modules;
+
+    /** The main coroutine for fiber execution. */
+    std::optional<::Fiber> main_co;
```

Comment:
We should move this into `Context` instead of global. Right now it won't make much of a difference (there's just one global context) but in the future, if/when we introduce threads, there'd be one context per thread, and we'd want to tie fibers to that.